src/index.ts

```diff
@@ -258,9 +259,9 @@ function init(modules: { typescript: typeof ts_module }) {
                   description: `Disable rule '${problem.getRuleName()}'`,
                   changes: [{
                     fileName: fileName,
-                    textChanges: [{

```

Comment:
No it's not a problem, I can accept your PR like this but the big pb is about compilation https://travis-ci.org/angelozerr/tslint-language-service/builds/225117180#L273 

Do you know how to fix this error?